Site/SiteAMQPModule.php

```diff
@@ -216,17 +222,17 @@ public function doSyncNs($namespace, $exchange, $message,
 		// callback function must return true or false in order to be handled
 		// correctly by the AMQP extension. If an exception is thrown in this
 		// callback, behavior is undefined.
-		$callback = function(AMQPEnvelope $envelope, AMQPQueue $queue)
-			use (&$response, $correlation_id)
-		{
+		$callback = function (AMQPEnvelope $envelope, AMQPQueue $queue)
+ use (&$response, $correlation_id) {
```

Comment:
This is an anonymous function that takes response and correlationid from the current scope. I think the original indentation was much better.